### PR TITLE
add redirects for deprecated CSP pages

### PR DESCRIPTION
**Pull request by tanyaparker**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Do we have a plan for how long we are going to maintain these redirects?

File: pegasus/sites.v3/code.org/public/educate/csp/unit4.moved [hunk 1]
```diff
@@ -0,0 +1 @@
+https://curriculum.code.org/csp/unit4v1/
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/csp/unit4.moved:1]: so v1 is our new naming standard for a while?

**Comment by breville** [comment]
lgtm

File: pegasus/sites.v3/code.org/public/educate/csp/unit4.moved (hunk 1 above)
Comment (tanyaparker) [inline pegasus/sites.v3/code.org/public/educate/csp/unit4.moved:1]: Units 4-6 are actually still temporary. I set a reminder to update these again to /csp/unit4 when those pages are launched (in a couple of weeks).

**Comment by tanyaparker** [comment]
@ashercodeorg what is a typical plan for keeping redirects? We have quite a lot that sit in our repo for a long time. Should we base it off traffic in GA, and if it's lower than X pageviews we can remove it?

I would want to eventually remove the unit redirects since those are pages that are replacing the old pages. But sometimes we use the redirect like code.org/widgets as a "short link" that's easier to share and remember. Should we be doing this differently?

**Pull request merged**